src/stats/TopProducts.php

```diff
@@ -22,6 +23,22 @@
  */
 class TopProducts extends Stat
 {
+    /**
+     * Constants for the type of stat query.
+     * @since 3.x
```

Comment:
@nfourtythree docblocks only apply to the thing that is defined directly after them. You can’t have one that acts as a heading for multiple things that follow it. So in this case `TYPE_QTY` is the only one that gets a description of “Costants for the type of stat query”, and “Since 3.x”.

Reply:
Good catch @brandonkelly these were placeholder for me to flesh out later and turns out I forgot they were there 🤦 

Will fix on the `3.4` branch